ext/enchant/enchant.stubs.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/** @return resource|false */
+function enchant_broker_init() {}
+
+function enchant_broker_free(string $broker): bool {}
```

Comment:
`$broker` has to be a resource, not a string. Since resource cannot be used as type hint, you can do something like:
```suggestion
/** @resource $broker */
function enchant_broker_free($broker): bool {}
```
Same for the following functions.

Reply:
Done!